@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -25,8 +25,19 @@ export interface ViewRanges {
      * @note Should be always positive and bigger then [[near]] plane distance.
      */
     far: number;
+    /**
+     * Minimum viewing distance that may be applied to near plane.
+     *
+     * @note Relfects minimum possible near plane distance regardles of camera and
+     * projection settings. Such constraint is always required because near plane
+     * can not be placed at zero distance from camera (regardless of rendering Api),
+     * moreover it may be used as input for algorighms based on visibility ranges
+     * such as it does not change during tilt.
+     */
+    minimum: number;
```

Comment:
I hope that is was explained few comments above. The near plane limit (minimum value) is usually dependent from the world scale. If your world is in units of millions it's reasonably to assume some number close to 1, but if you shrink to world to 0-1 then it need to be really small, otherwise you loose you geometry in front of camera. Here the defaults suits our projections, but if used provide its own, it may be totally different - thus configurable.